FAQ.md

```diff
@@ -0,0 +1,114 @@
+# FAQ
+
+Maintained by @eseidel
+
+If you have questions not answered here, please ask (either by filing an issue
+or asking on [Discord](https://discord.gg/9hKJcWGcaB)).
+
+# What is "code push"?
+
+Code push, also referred to as "over the air updates" (OTA) is a cloud service
+we are building to enable Flutter developers to deploy updates directly to
+devices anywhere they have shipped Flutter.  We've currently shown demos using
+Android, but since it's built with Flutter/Dart (and Rust) it will work anywhere
+Flutter works.
+
+The easiest way to see is to watch the [demo
+video](https://www.youtube.com/watch?v=mmKvs0_Zu14&ab_channel=Shorebird).
+
+Code push is a working title.  We'll probably come up with some name to help
+differentiate it.  "Code Push" is a reference to the name of a deploy feature
+used by the React Native community from [Microsoft](appcenter.ms) and
+[Expo](appcenter.ms), neither of which support Flutter.
```

Comment:
Do you mean https://appcenter.ms ? Currently the link returns a 404 from github.